jobs/aide/templates/bin/run-report.erb

```diff
@@ -6,7 +6,13 @@ JOB_DIR=/var/vcap/jobs/aide
 PACKAGE_DIR=/var/vcap/packages/aide
 LOGFILE=/var/vcap/sys/log/aide/report.log
 
-$PACKAGE_DIR/bin/aide --check > report.txt
+pushd "${JOB_DIR}/bin"
+  source lock-functions
+popd
```

Comment:
Is the pushd/popd needed. Can we just `source ${JOB_DIR}/lock-functions`?

Reply:
yeah, that's probably better